data/inspector/main.js

```diff
@@ -91,12 +96,12 @@ var actions = {
  */
 var content = document.getElementById("content");
 var theApp = React.render(MainTabbedArea({
-  packets: [],
   actions: actions
 }), content);
 
 // Helper modules for handling application events.
-var store = new PacketsStore(window, theApp);
+var packetsStore = new PacketsStore(window, theApp);
+var actorsStore = window.actorsStore = new ActorsStore(window, theApp);

```

Comment:
@janodvarko removed in cc38091 (sorry, I forgot to remove it, I used it to inject data to speed-up the bootstrap of the React components bootstrap)